web-client/src/presenter/computeds/scanBatchPreviewerHelper.test.js

```diff
@@ -0,0 +1,82 @@
+import { Scan } from '../../../../shared/src/business/entities/Scan';
+import { User } from '../../../../shared/src/business/entities/User';
+import { applicationContext } from '../../applicationContext';
+import { runCompute } from 'cerebral/test';
+import { scanBatchPreviewerHelper as scanBatchPreviewerHelperComputed } from './scanBatchPreviewerHelper';
+import { withAppContextDecorator } from '../../withAppContext';
+
+const state = {
+  batches: [],
+};
+
+const { SCAN_MODES } = Scan;
+
+const scanBatchPreviewerHelper = withAppContextDecorator(
+  scanBatchPreviewerHelperComputed,
+  applicationContext,
+);
+
+applicationContext.getCurrentUser = () => ({ role: User.ROLES.practitioner });
+applicationContext.getConstants = () => ({ SCAN_MODES });
+
+describe('scanBatchPreviewerHelper', () => {
+  beforeEach(() => {
+    state.form = {};
+  });
+
+  describe('scannerSourceDisplayName', () => {
+    it('returns correct values when no scanner is selected', () => {
+      let testState = { ...state };
+
+      const result = runCompute(scanBatchPreviewerHelper, {
+        state: testState,
+      });
+      expect(result.scannerSourceDisplayName).toEqual('None');
+    });
+
+    it('returns correct values when a scanner is selected and is using single sided', () => {
```

Comment:
Where here is it being established that it's single-sided?

Reply:
Single sided is the default. The test description has been updated.